test/mocha/20-tokens.js

```diff
@@ -770,6 +775,141 @@ describe('Tokens Database Tests', function() {
       executionStats.totalDocsExamined.should.equal(1);
       executionStats.executionStages.inputStage.inputStage.inputStage.stage
         .should.equal('IXSCAN');
+      executionStats.executionStages.inputStage.inputStage.inputStage.keyPattern
+        .should.eql({
+          'pairwiseToken.internalId': 1, 'pairwiseToken.requester': 1
+        });
+    });
+  });
+});
+
+describe('Entities Database Tests', function() {
+  describe('Indexes', function() {
+    beforeEach(async () => {
+      const collectionName = 'tokenization-entity';
+      await cleanDB({collectionName});
+
+      mockData.mockEntity1.entity.internalId = Buffer.from(await generateId());
```

Comment:
`generateId` returns a string, so this is creating a Buffer from a string -- we don't want that. We want the non-encoded bytes of the identifier.

You'll need to import `IdGenerator` from `bnid` instead and call `generate()` on an instance of it. Something like this:

```suggestion
      const idGenerator = new IdGenerator();
      mockData.mockEntity1.entity.internalId = Buffer.from(await idGenerator.generate());
```